sdk/src/client/secret/ledger_nano.rs

```diff
@@ -179,11 +242,6 @@ impl SecretManageExt for LedgerSecretManager {
             });
         }
 
-        if coin_type.is_none() || account_index.is_none() {
-            return Err(InputSelectionError::NoAvailableInputsProvided)?;
-        }
```

Comment:
This should never happen anyway?

Reply:
Well yes it shouldn't happen, but since the values are optional it can happen and then it would panic if we don't return an error before

Reply:
But I mean, we already return an error if it's None on L227 so this is never going to happen ?

Reply:
Meh, I'll revert this then

Reply:
Done